api/api.proto

```diff
@@ -617,10 +639,38 @@ message LinkFacebookRequest {
   google.protobuf.BoolValue import = 4;
 }
 
-// A collection of zero or more friends of the user.
-message Friends {
-  // The Friend objects.
-  repeated Friend friends = 1;
+// Fetch a list of unexpired notifications.
+message ListNotificationsRequest {
+  // Max number of notifications to list. Between 1 and 100.
+  int64 limit = 1;
+  // Use this cursor to paginate notifications. Cache this to catch up to new notifications. The value of this comes from NotificationList.cachable_cursor.
+  string cachable_cursor = 2;
```

Comment:
Let's use `cacheable` as the spelling, seems to be the more standard form used in W3C texts etc